code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -0,0 +1,163 @@
+//Xenobio control console
+/mob/camera/aiEye/remote/xenobio
+	visible_icon = 1
+	icon = 'icons/obj/abductor.dmi'
+	icon_state = "camera_target"
+
+
+/mob/camera/aiEye/remote/xenobio/setLoc(var/t)
+	var/area/new_area = get_area(t)
+	if(new_area.name && new_area.name == "Xenobiology Lab")
```

Comment:
Yeah but doing it by name allows for territory expansion via blueprint use so you can turn the game into dungeon keeper/zombie master and drop slimes on everyone until they come murder you